test/test_type.jl

```diff
@@ -109,7 +109,7 @@ isdir(outdir) && rm(outdir, recursive = true)
 
             x = SVector(zero(RealT))
             t = zero(RealT)
-            u = u_ll = u_rr = u_inner = SVector(one(RealT), one(RealT), one(RealT))
+            u = u_ll = u_rr = u_inner = SVector(one(RealT), zero(RealT), zero(RealT))
```

Comment:
I wonder why you changed the values here. Could you please explain this?

Reply:
Boring reasons. `CompressibleNavierStokesDiffusion` is related to `CompressibleEulerEquations`, and the old `u` value led to complex value in a bunch of tests for the former equation and the new `u` could succeed. So just change them together.

Do you want me to change it back for it?

Reply:
No, that's fine. I was just wondering. Thanks for the explanation 👍